pkg/policy/api/rule_validation.go

```diff
@@ -120,6 +121,10 @@ func (i *IngressRule) sanitize() error {
 		return fmt.Errorf("The ICMPs block may only be present without ToPorts. Define a separate rule to use ToPorts.")
 	}
 
+	if len(i.FromNodes) > 0 && !option.Config.EnablePerNodeIdentity {
```

Comment:
We need to check `PerNodeIdentitiesEnabled` here as well. You can change the error message below as well if you like, but I don't think it's necessary.
```suggestion
	if len(i.FromNodes) > 0 && (!option.Config.PerNodeIdentitiesEnabled() || 
	!option.Config.EnablePerNodeIdentity) {
```

Reply:
both are the same thing.. guess I could just directly access `option.Config.EnablePerNodeIdentity` in all the cases?